UKHomeOffice/nmsw-ui
# NMSW-466 Update govuk logo to take user to https://www.gov.uk/

# Ticket

NMSW-466

----

## AC

Currently when you click on the GOV logo it takes you to the NMSW landing page. However, the GOV logo should actually take you to https://www.gov.uk/ (Confirmed with service designer)

----

## To test

- Run app
- Go to localhost:3000
- Click on the GOV logo
- It should take you to https://www.gov.uk/

----

## Notes
